src/sidebar/util/immutable.js

```diff
@@ -25,6 +25,9 @@ function deepFreeze(object) {
  * @return {Object} Returns the input object
  */
 export default function immutable(object) {
+  // TESTING
+  return object;
```

Comment:
This is just me debugging an apparent performance issue with this function. Please ignore.